examples/pennylane_skip_plugins_hybrid.py

```diff
@@ -0,0 +1,391 @@
+r"""
+.. _plugins_hybrid:
+
+Plugins and Hybrid computation
+==============================
+
+This tutorial introduces the notion of hybrid computation by combining several PennyLane
+plugins. We first introduce PennyLane's `Strawberry Fields plugin <https://github.com/XanaduAI/pennylane-sf>`_
+and use it to explore a non-Gaussian photonic circuit. We then combine this photonic circuit with a
+qubit circuit — along with some classical processing — to create and optimize a fully hybrid computation.
+Be sure to read through the introductory :ref:`qubit rotation <qubit_rotation>` and
+:ref:`Gaussian transformation <gaussian_transformation>` tutorials before attempting this tutorial.
+
+.. note::
+
+    To follow along with this tutorial on your own computer, you will require the
+    `PennyLane-SF plugin <https://github.com/XanaduAI/pennylane-sf>`_, in order to access the
+    `Strawberry Fields <https://github.com/XanaduAI/strawberryfields>`_ Fock backend using
```

Comment:
same here: should we point to readthedocs?

Reply:
as soon as we've decided I'll update everything to either github or rtd

Reply:
Done